### DISCUSSION TITLE
Dokumentation falsch - Huawei AC charger topics

### DISCUSSION THREAD

**Discussion by Phantomias2006** [hoylabs/OpenDTU-OnBattery#312] [General]
Guten Tag,

in der Dokumentation ist beschrieben wie man das Huawei AC Netzteil über MQTT On/Off schalten kann. Da dies bei mir nicht funktioniert hat habe ich mir den SourceCode dazu angesehen und musste feststellen dass sich das Topic geändert hat.

In der Doku:
`huawei/cmd/power`
Im SourceCode:
`huawei/cmd/mode`

Weiters gibt es jetzt 4 statusse:

- 0:Turned OFF
- 1:Turned ON
- 2:Internal on/off control, external power limit
- 3:Full internal control 

Kann jemand die Doku diesbezüglich anpassen?
Besten Dank
![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/15437240/85e5de2f-b425-433c-8098-b3f5c7779033)

**Comment by madmartin** [comment]
Hi, 
danke für's melden!

Machst Du einen Pull-Request dazu?
Wenn Du das nicht kannst kann ich das auch machen.

**Reply by Phantomias2006** [reply]
Gerne :-)
https://github.com/helgeerbe/OpenDTU-OnBattery/pull/313

LG

**Comment by Phantomias2006** [comment]
@madmartin 
Was mir noch aufgefallen ist:
Das Topic "battery" ist Camel Case und das Topic "huawei" ist Snake Case.
LG
